src/program/config/common.lua

```diff
@@ -73,37 +73,58 @@ function parse_command_line(args, opts)
    function handlers.s(arg) ret.schema_name = arg end
    function handlers.r(arg) ret.revision_date = arg end
    function handlers.c(arg) ret.socket = arg end
+   function handlers.v(arg) ret.verbose = true end
    function handlers.f(arg)
       assert(arg == "yang" or arg == "xpath", "Not valid output format")
       ret.format = arg
    end
    handlers['print-default'] = function ()
       ret.print_default = true
    end
-   args = lib.dogetopt(args, handlers, "hs:r:c:f:",
-                       {help="h", ['schema-name']="s", schema="s",
+   args = lib.dogetopt(args, handlers, "hvs:r:c:f:",
+                       {help="h", verbose="v", ['schema-name']="s", schema="s",
                         ['revision-date']="r", revision="r", socket="c",
                         ['print-default']=0, format="f"})
+
    if #args == 0 then err() end
-   ret.instance_id = table.remove(args, 1)
-   local descr = call_leader(ret.instance_id, 'describe', {})
-   if not ret.schema_name then
-      if opts.require_schema then err("missing --schema arg") end
-      ret.schema_name = descr.default_schema
+
+   if opts.command == 'test' then
```

Comment:
I dunno, I think since there's these two ways to deal with the schema -- either as provided by the user or as received from the remote -- I think it makes sense to move all argument handling to `test.lua`, to avoid making this `common.lua` too specific on the commands being used.  WDYT?